sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -337,7 +337,17 @@ export class CallbackFunction<E, R> extends LambdaFunction {
             closure, serializedFileNameNoExtension, args.codePathOptions);
 
         const code = pulumi.output(new pulumi.asset.AssetArchive(codePaths));
-        (<any>code).isSecret = closure.then(c => c.containsSecrets);
+        (<any>code).isSecret = closure.then(c => {
+            if (c.containsSecrets) {
+                pulumi.log.warn(`A secret value was captured and serialized into the body of the Lambda Function '${name}'. ` +
+                                `This value will be stored as an encyrpted Pulumi secret, but may be available in plain text ` +
+                                `inside the AWS deployment package.  You can use 'pulumi.unsecret' to convert the value to a non-secret ` +
+                                `value if this is not a sensitive value, or else use Secrets Manager or environment variables to pass the` +
```

Comment:
We just need one space here, I think:

```suggestion
                                `value if this is not a sensitive value, or else use Secrets Manager or environment variables to pass the ` +
```

For reference here's how this renders out:

![image](https://user-images.githubusercontent.com/274700/121562052-ef961080-c9cd-11eb-9343-8df8f1ca7900.png)